master/internal/rm/kubernetesrm/pods.go

```diff
@@ -1568,7 +1568,11 @@ func (p *pods) getNonDetPods() []k8sV1.Pod {
 		return nonDetPods
 	}
 	for _, p := range pList.Items {
-		if _, ok := p.Labels["determined"]; !ok {
+		_, ok1 := p.Labels[determinedLabel]
+		_, ok2 := p.Labels[determinedSystemLabel]
+		_, ok3 := p.Labels[determinedPreemptionLabel]
```

Comment:
I would name these differently, personally.
```suggestion
		_, isDet := p.Labels[determinedLabel]
		_, isDetSystem := p.Labels[determinedSystemLabel]
```